libs/platform/ebpf_epoch.c

```diff
@@ -384,6 +380,14 @@ _ebpf_epoch_release_free_list(_In_ ebpf_epoch_cpu_entry_t* cpu_entry, int64_t re
             break;
         }
     }
+    // If there are still items in the free list, schedule a timer to reap them in the future.
+    if (!ebpf_list_is_empty(&cpu_entry->free_list) && !cpu_entry->flags.timer_armed) {
+        // We will arm the timer once per CPU that sees entries it can't release.
+        // That's acceptable as arming the timer is idempotent.
+        cpu_entry->flags.timer_armed = true;
+        ebpf_schedule_timer_work_item(_ebpf_flush_timer, EBPF_EPOCH_FLUSH_DELAY_IN_MICROSECONDS);
```

Comment:
Now multiple timer routines can run in parallel on several cores. How does that impact ebpf_epoch_flush? 
```
    if (return_value == EBPF_SUCCESS) {
        _ebpf_release_epoch = released_epoch;
    }
    ```

Reply:
There is only one timer. It will be armed up to cpu_count times and will fire EBPF_EPOCH_FLUSH_DELAY_IN_MICROSECONDS  after the last CPU arms it.

released_epoch is only written to by the timer, so no interlocked operation is required.